src/app/(dashboard)/attestation/utils/attestation-utils.ts

```diff
@@ -84,6 +106,78 @@ export async function createPrivateAttestation({
   return { attestationUuid: attestationUuid };
 }
 
+export async function createPublicAttestation({
+  address,
+  data,
+  signer,
+  chainId,
+}: CreatePublicAttestationBodyDto): Promise<AttestationUuidDto | null> {
+  const chainConfig = chainsConfig[chainId];
+  if (!chainConfig) {
+    console.error(`No config found for chain id: ${chainId}`);
+    return null;
+  }
+
+  const schemaUID = chainConfig.publicAttestationSchemaUUId;
+  if (!schemaUID) {
+    console.error(
+      `No public attestation Schema found for chain id: ${chainId}`,
+    );
+    return null;
+  }
+
+  const eas = await initializeEAS(signer, chainConfig);
+  if (eas == null) {
+    return null;
+  }
+
+  const schemaEncoder = new SchemaEncoder(
+    "string githubUsername,string measuredAt,string repositoryName,string organizationName,string prNodeWeight,string prReviewNodeWeight,string issueNodeWeight,string commentNodeWeight,string commitNodeWeight,string userScoreRank,string userCredScore,string userCredScorePercentage",
+  );
+  const encodedData = schemaEncoder.encodeData([
+    { name: "githubUsername", value: data.githubUsername, type: "string" },
+    { name: "measuredAt", value: data.measuredAt, type: "string" },
+    { name: "repositoryName", value: data.repositoryName, type: "string" },
+    { name: "organizationName", value: data.organizationName, type: "string" },
+    { name: "prNodeWeight", value: data.prNodeWeight, type: "string" },
```

Comment:
Let's use the actual weight (0.025, 0.5, etc instead of the slider values on the weights)

Reply:
This is already the case

Reply:
The same data that is used for private attestation is used for public